elementor/elementor
# Fix: Whatsapp share link does not escape html entities (PT 553)

## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Fix: Whatsapp share link does not escape html entities

## Description
An explanation of what is done in this PR

* Added HTML entity decoding to all page titles, and added URL encoding to make sure WhatsApp handles special characters properly

## Test instructions
This PR can be tested by following these steps:

- Create a new page, and give the page a title with special characters like quotes and ampersands (`&`).
- Edit the page with Elementor
- Add a Share Buttons widget
- Add Share Buttons for all platforms, share the page and see that the page title and page URL are displayed correctly in the shared post
